scipy/stats/stats.py

```diff
@@ -1180,6 +1180,41 @@ def kurtosis(a, axis=0, fisher=True, bias=True, nan_policy='propagate'):
     >>> from scipy.stats import kurtosis
     >>> kurtosis([1, 2, 3, 4, 5])
     -1.3
+
+    In Fisher's definiton, the kurtosis of the normal distribution is zero.
+    In the following example, the kurtosis is close to zero, becuase it was
+    calculated from the dataset, not from the continuous distribution.
+
+    >>> from scipy.stats import norm
+    >>> data = norm.rvs(size=1000)
+    >>> kurtosis(data)
```

Comment:
This will return random output and it will trip up the ref guide checks as did now on Travis.